docs/rules/max-attributes-per-line.md

```diff
@@ -0,0 +1,88 @@
+# Define the number of attributes allows per line (max-attributes-per-line)
+
+Limits the maximum number of props per line to improve readability
+
+
+## Rule Details
+
+This rule aims to enforce a number of props per line in templates. It checks all the elements in a template and verifies that the number of props per line does not exceed the defined maximum.
+A prop is considered to be in a new line when there is a line break between two props.
```

Comment:
Maybe something like: `There is predefined number of props that are acceptable in one-line case, as well as how many attributes are acceptable per line in multi-line case.`?